backend/oko_pg_interface/scripts/bin/migrate.sql

```diff
@@ -106,10 +106,11 @@ CREATE INDEX idx_email_verifications_expires ON public.email_verifications USING
 CREATE TABLE public.ewallet_users (
 	user_id uuid DEFAULT gen_random_uuid() NOT NULL,
 	email varchar(255) NOT NULL,
+	vendor varchar(64) DEFAULT 'google'::character varying NOT NULL,
```

Comment:
migrate.sql will be deprecated. You don't have to edit this file anymore

Reply:
I think it’s still useful to keep migrate.sql as a reference for the latest full schema.